packages/lib/EventManager.ts

```diff
@@ -447,12 +447,34 @@ export default class EventManager {
     }
 
     const referencesToCreate = results.map((result) => {
+      // For update operations, check updatedEvent first, then fall back to createdEvent
+      const updatedEvent = Array.isArray(result.updatedEvent) ? result.updatedEvent[0] : result.updatedEvent;
+      const createdEvent = result.createdEvent;
+      let event = updatedEvent;
+      if (!event) {
+        log.warn(
+          "updateLocation: No updatedEvent when doing updateLocation. Falling back to createdEvent but this is probably not what we want",
+          safeStringify({ bookingId: booking.id })
+        );
```

Comment:
We usually do fallback at ohehr places. So doing the same, but adding a warning to see if it occurs or not. I think it shouldnt happen.